app/Http/Controllers/AppealController.php

```diff
@@ -112,15 +113,47 @@ public function appeallist()
                 });
         }
 
-        $hiddenStatuses = $isDeveloper
-            ? [Appeal::STATUS_ACCEPT, Appeal::STATUS_DECLINE, Appeal::STATUS_EXPIRE, Appeal::STATUS_INVALID]
-            : [Appeal::STATUS_ACCEPT, Appeal::STATUS_DECLINE, Appeal::STATUS_EXPIRE, Appeal::STATUS_VERIFY, Appeal::STATUS_NOTFOUND, Appeal::STATUS_INVALID];
+        $appealtypes = ['assigned'=>'Assigned to me','unassigned'=>'All unreserved open appeals','reserved'=>'Open reserved appeals'];
+        if($isDeveloper) { $appealtypes['developer']='Developer access appeals'; }
 
-        $appeals = Appeal::whereIn('wiki', $wikis)
-            ->whereNotIn('status', $hiddenStatuses)
+        $developerStatuses = [Appeal::STATUS_VERIFY, Appeal::STATUS_NOTFOUND];
+        $basicStatuses = [Appeal::STATUS_ACCEPT, Appeal::STATUS_DECLINE, Appeal::STATUS_EXPIRE, Appeal::STATUS_VERIFY, Appeal::STATUS_NOTFOUND, Appeal::STATUS_INVALID, Appeal::STATUS_CHECKUSER];
+
+        $appeals[$appealtypes['assigned']] = Appeal::whereIn('wiki', $wikis)->where(function ($query) use ($basicStatuses) {
+            $query->whereNotIn('status', $basicStatuses)
+            ->where('handlingadmin',Auth::id());
+        })->orWhere(function ($query) use ($isCUAnyWiki) {
+            if ($isCUAnyWiki) {
+                $query->where('status',Appeal::STATUS_CHECKUSER);
+            }
```

Comment:
Is it intentional that CUs can only see CU appeals?

Reply:
As noted by the variable, `$appeals[$appealtypes['assigned']]`, the `assigned` parts means they need to deal with it. CUs need to deal with the CU appeals. Non CUs don't. See where you comment below, they show up elsewhere.

Reply:
Yes. bit this will *prevent* CUs from dealing for non-CU appeals